### PR TITLE
Feat/unify nuxt dependencies

### LINKED ISSUE #18: @nuxtjs/composition-api is always required
Hi,

In the documentation, it seems you are giving us the choice to use the plugin with Composition API or not.
Then, the `@nuxtjs/composition-api` plugin should only be required if needed.

But currently, it's a basic requirement; a project using the `storyblok-nuxt-2` plugin without `@nuxtjs/composition-api` won't start.

Screenshot 2022-06-17 at 15 16 23@2x

Would it be possible to fix that so that we are not required to add `@nuxtjs/composition-api` in our build?
It has a signifiant impact on build size.

Reproduction link: https://codesandbox.io/s/green-sun-45ptuw?file=/nuxt.config.js

Thanks!

### PR DESCRIPTION
**Pull request by alexjoverm**

### CODE REVIEW

File: playground/package.json [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate"
   },
-  "dependencies": {
-    "@nuxtjs/composition-api": "^0.32.0",
+  "devDependencies": {
     "@storyblok/nuxt-2": "^0.0.1",
-    "core-js": "^3.22.2",
+    "core-js": "^3.15.1",
```
Comment (manuelschroederdev) [inline playground/package.json]: Just curious, why did you have to downgrade this?

File: playground/nuxt.config.js [hunk 2]
```diff
@@ -12,11 +12,10 @@ export default {
       { name: "format-detection", content: "telephone=no" },
     ],
     link: [{ rel: "icon", type: "image/x-icon", href: "/favicon.ico" }],
-    script: [{ src: "https://cdn.tailwindcss.com" }],
   },
 
   buildModules: [
     ["@storyblok/nuxt-2/module", { accessToken: "OurklwV5XsDJTIE1NJaD2wtt" }],
-    "@nuxtjs/composition-api/module",
```
Comment (manuelschroederdev) [inline playground/nuxt.config.js:20]: Very cool to see this getting redundant 👏

**Review by manuelschroederdev — APPROVED** [review]
Fantastic work, Alex! 🚀
Everything looks good from my point of view.

File: playground/package.json (hunk 1 above)
Comment (alexjoverm) [reply playground/package.json]: Only life knows... fixing it ;)

**Pull request merged**